src/xrpld/app/misc/NetworkOPs.cpp

```diff
@@ -1204,15 +1214,25 @@ NetworkOPsImp::processTransaction(
         return;
     }
 
+    auto const view = m_ledgerMaster.getCurrentLedger();
+
+    // This function is called by several different parts of the codebase
+    // under no circumstances will we ever accept an inner txn within a batch
+    // txn from the network.
+    auto const tx = *transaction->getSTransaction();
```

Comment:
```suggestion
    auto const sttx = *transaction->getSTransaction();
```
consider renaming this since `tx` conflicts with `transaction`